guides/video_pagination.md

```diff
@@ -0,0 +1,138 @@
+# Video Pagination with Active Speaker-Based Policy
+
+## Overview
+
+Amazon Chime SDK currently supports up to 16 concurrent video streams in a meeting. However, in many cases, the application may not want to render all the available video streams, because of the following reasons:
+
+* **Hardware limitation** - On mobile devices, screen size is relatively small and network as well as computation resources are rather limited. If an application renders all the available videos on the same screen at the same time, it will consume a lot of network bandwidth and CPU to subscribe to video streams and decode video frames, but each video tile will be extremely small and barely visible, which results in a bad user experience.
```

Comment:
`screen sizes are`?
Start a new sentence with `Each video tile will...`. It's quite long currently.